internal/pkg/describe/describe.go

```diff
@@ -38,3 +39,7 @@ func NewAppIdentifier(project, app string) (ResourceIdentifier, error) {
 type stackDescriber interface {
 	DescribeStacks(input *cloudformation.DescribeStacksInput) (*cloudformation.DescribeStacksOutput, error)
 }
+
+type sessionFromRoleProvider interface {
```

Comment:
seems like this interface is dup in `internal/pkg/cli/cli.go` as well?

Reply:
yeah I'm not sure what would be a good place to put it :| I want each package to define their own dependency. That's why the interface is replicated. Do you have any recommendations?

Reply:
I'm gonna go ahead and merge but let me know! I didn't want to put it in the `aws/session` pkg because it's not using the interface